lib/sprite.dart

```diff
@@ -1,20 +1,21 @@
 import 'dart:ui';
 
+import 'extensions/offset.dart';
 import 'extensions/vector2.dart';
 import 'palette.dart';
 
 class Sprite {
   Paint paint = BasicPalette.white.paint;
   Image image;
-  Rect src;
+  Rect bounds;
 
   Sprite(
     this.image, {
     Vector2 position,
     Vector2 size,
   }) : assert(image != null, "image can't be null") {
     size ??= Vector2(image.width.toDouble(), image.height.toDouble());
-    src = position.toPositionedRect(size);
+    this.position = position;
```

Comment:
`?? Vector.empty()`? or you can add the default value to the constructor param, either works

Reply:
It is done in the setter